parachains/runtimes/bridge-hubs/README.md

```diff
@@ -191,65 +195,10 @@ RUST_LOG=runtime=trace,rpc=trace,bridge=trace \
 	- Pallet: **bridgeRococoParachain**
 	- Keys: **bestParaHeads()**
 
-### Send messages
-
-#### Local zombienet run
```

Comment:
These instructions are useful for setting up local testing, why remove them (I don't see them in #2013 README.md so looks like they are lost if we remove here)?

Reply:
those are not actual and I will add them for ro/wo in new branch `origin/bko-transfer-asset-via-bridge-pallet-xcm-ro-wo` with `pallet_xcm` support for transfer